src/autocomplete/autocomplete.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from trie import Trie
+
+
+class AutoCompleter(object):
+    """Python implementation of autocomplete class."""
+    def __init__(self, vocabulary, max_completions=5):
+        """Initialize the Autocompleter class."""
+        self.vocabulary = Trie(iterable=vocabulary)
+        self.max_completions = max_completions
+
+    def __call__(self, token):
+        """Takes user input and returns list of suggested words, max five."""
+        suggested_words = []
+        if not isinstance(token, str):
+            return suggested_words
+        start = self.vocabulary.root
+        token = ""
+        completions = 0
+        while completions < self.max_completions:
+            while True:
+                if start.keys()[0] != '#':
```

Comment:
Your code must be compatible with both Python 2 and Python 3.

In Python 3, `dict.keys()` and `OrderedDict.keys()` returns a generator, not a list, and a generator does not support indexing.